index.html

```diff
@@ -1,188 +1,202 @@
-                        <li><a href="#"><img class="icon" src="img\peace icon.svg" alt="peace icon"></a></li>
-                    </ul>
-                    <button class="button">Get my resume</button>
+                <p>
+                    Hello I’m a software developer! I can help
+                    you build a product , feature or website
+                    Look through some of my work and experience!
+                    If you like what you see and have a project you
+                    need coded, don’t hesitate to contact me.
+                </p>
+                <h3>LET’S CONNECT</h3>
+                <ul class="socialIcon">
+                    <li><a href="#"><img class="icon" src="img\Vector.svg" alt="Tweeter icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\Linkedin icon.svg" alt="Linkedin icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\Vector m.svg" alt="Mail icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\github icon.svg" alt="Github icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\peace icon.svg" alt="Peace icon"></a></li>
+                </ul>
+                <button class="button">Get my resume</button>
             </section>
             <section class="languages">
                 <article class="dropdownHead" id="open">
                     <h4>Languages</h4>
-                    <img src="img\Union.svg">
+                    <img src="img\Union.svg" alt="Vector icon">
                 </article>
                 <ul class="listOfLanguages">
-                    <li><img src="img\javaicon.svg" alt="javaScript"><p>JavaScript</p></li>
-                    <li><img src="img\htmlicon.svg" alt="HTML"><p>HTML</p></li>
-                    <li><img src="img\cssicon.svg" alt="CSS"><p>CSS</p></li>
+                    <li><img src="img\javaicon.svg" alt="javaScript">
+                        <p>JavaScript</p>
+                    </li>
+                    <li><img src="img\htmlicon.svg" alt="HTML">
+                        <p>HTML</p>
+                    </li>
+                    <li><img src="img\cssicon.svg" alt="CSS">
+                        <p>CSS</p>
+                    </li>
                 </ul>
                 <article class="dropdownHead">
                     <h4>Frameworks</h4>
-                    <img src="img\Unionside.svg">
+                    <img src="img\Unionside.svg" alt="Side vector icon">
                 </article>
                 <article class="dropdownHead">
                     <h4>Skills</h4>
-                    <img src="img\Unionside.svg">
+                    <img src="img\Unionside.svg" alt="Side vector icon">
                 </article>
             </section>
         </section>
     </main>
     <section class="contact">
         <h1>Contact me</h1>
```

Comment:
- [ ] The page has only one heading `<h1>` please kindly change your `h1` to another `hx` tag. `eg: <h2>Contact me</h2>` 👍 👍